webapp/javascript/components/TimelineChart/TimelineChartWrapper.tsx

```diff
@@ -193,29 +202,28 @@ class TimelineChartWrapper extends React.Component<
       // to use custom apperance and color for it
       const boundary = {
         lineWidth: 1,
-        color:
-          this.props.selectionType === 'double' ? m.color.rgb() : 'transparent',
+        color: m.color.rgb(),
       };
 
       return [
         {
           xaxis: { from, to },
           color:
-            this.props.selectionType === 'double'
-              ? m.overlayColor
-              : 'tranparent',
+            this.props.selectionType === 'double' ? m.overlayColor : 'NOOP',
```

Comment:
what the purpose of setting NOOP here?

Reply:
`noop` stands for "No operation", I am doing it here since `tranparent` is not a valid color, hence `NOOP` to make it clear it's something NOT valid